packages/common/utils/src/clamp.spec.ts

```diff
@@ -1,17 +1,29 @@
 import { clamp } from './clamp';
 
-describe('clamp는', () => {
-  it('max 만 있을 때 잘 동작한다', () => {
+describe('clamp should', () => {
```

Comment:
it would be better to write `"should"` in `it()`:
```typescript
describe("clamp", () => {
  it("should work ...", () => {
    /* ... */
  });
});
```
so that developers can read straight.

Reply:
They are functionally the same but semantically different.

```typescript
it("should have a button", () => { ... }); // 1
test("should have a button", () => {}); // 2
```

you can read the code above as english sentence and then what's the more easily readable? I think `1` is the winner.

Reply:
I didn't immediately recognize the difference after seeing the comments you wrote. After reading the comments about three times, I felt exactly meaning you wrote 'semantically different', so I realized exactly that the method should be it not test. As a front-end developer, I didn't realize the difference between it and test when I wrote the test code, and I didn't get such a clear answer anywhere, but I finally got a clear answer tonight. Thanks a lot. I'll reflect on it.